docs/src/main/sphinx/installation/cli.rst

```diff
@@ -358,8 +364,51 @@ The following table list the available options for Kerberos authentication:
   * - ``--krb5-service-principal-pattern``
     - Remote kerberos service principal pattern (default: ${SERVICE}@${HOST})
 
-See :doc:`/security/cli` for more information on configuring and using Kerberos
-with the CLI.
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+Many of the same steps, that can be used when troubleshooting the :ref:`Trino
+coordinator <coordinator-troubleshooting>` apply to troubleshooting the CLI.
+
+Additional Kerberos debugging information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can enable additional Kerberos debugging information for the Trino CLI
+process by passing ``-Dsun.security.krb5.debug=true``,
+``-Dtrino.client.debugKerberos=true``, and
+``-Djava.security.debug=gssloginconfig,configfile,configparser,logincontext``
+as a JVM argument, when starting the CLI process. Doing so requires invoking
+the CLI JAR via ``java`` instead of running the self-executable JAR directly.
+
+.. code-block:: text
+
+    #!/bin/bash
+
+    java \
+      -Dsun.security.krb5.debug=true \
+      -Djava.security.debug=gssloginconfig,configfile,configparser,logincontext \
+      -Dtrino.client.debugKerberos=true \
+      -jar trino-cli-*-executable.jar \
+      --server https://trino-coordinator.example.com:7778 \
+      --krb5-config-path /etc/krb5.conf \
+      --krb5-principal someuser@EXAMPLE.COM \
+      --krb5-keytab-path /home/someuser/someuser.keytab \
+      --krb5-remote-service-name trino \
+      --keystore-path /tmp/trino.jks \
+      --keystore-password password \
+      --catalog <catalog> \
+      --schema <schema>
+
+The :ref:`additional resources <server_additional_resources>` listed in the
```

Comment:
Need to check what we link to and reword

Reply:
This links to the debugging section in the Kerberos authentication doc.